DigitalLearningSolutions.Data.Migrations/202201111458_LearningResourceReferencesOriginalRatingField.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+    [Migration(202201111458)]
+    public class LearningResourceReferencesOriginalRatingField : Migration
+    {
+        public override void Up()
+        {
+            Alter.Table("LearningResourceReferences")
+                .AddColumn("OriginalRating").AsDecimal(2, 1).Nullable();
```

Comment:
The open API will always return a non-null value for this, so I'd expect this column to be non-null.
I missed this for the previous migration, but that's also true for the other columns.

Are there reasons you would prefer the columns to be nullable?

Reply:
Laziness really. It is simply to avoid having to delete existing records in all of the related tables in dev database (because we are actively working with that data). Will nullable fields cause problems?

Reply:
I think with decimals having a non-null column would make a significant difference to the code quality, as it would remove unexplained assumptions that rating will be non-null. You could use `WithDefaultValue()` to set the default value as 0 in the migration, which would save you deleting entries, and would also get you safer test data.
I'm a bit less concerned about string fields because we're going to treat whitespace strings as if they were null anyway.

Reply:
Good idea. Updated.